src/NServiceBus.AcceptanceTests/Routing/When_publishing_pre_created_interface.cs

```diff
@@ -0,0 +1,115 @@
+﻿namespace NServiceBus.AcceptanceTests.Routing
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using Features;
+    using NServiceBus.Pipeline;
+    using NUnit.Framework;
+    using Conventions = AcceptanceTesting.Customization.Conventions;
+
+    public class When_publishing_pre_created_interface : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_receive_event()
```

Comment:
maybe something like (not sure whether this is already too long):
```suggestion
        public async Task Should_publish_event_to_interface_type_subscribers()
```